src/raftstore/store/fsm/store.rs

```diff
@@ -585,6 +595,28 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         Ok(true)
     }
 
+    // check whether the range is covered by exist regions.
+    fn is_range_covered(&self, start: Vec<u8>, end: Vec<u8>) -> bool {
+        if let Some((_, id)) = self
+            .region_ranges
+            .range((Included(end), Unbounded::<Key>))
```

Comment:
Why not search with start key? So that only need to check whether all following regions are continuous and cover [start_key, end_key), which seems more straightforward.